content/doc/book/pipeline/shared-libraries.adoc

```diff
@@ -159,6 +157,10 @@ link:http://groovy-lang.org/objectorientation.html#_annotation[annotation]
 pattern `@Library('my-shared-library') _` may be useful for keeping code
 concise. In essence, instead of annotating an unnecessary `import` statement,
 the symbol `_` is annotated.
+
+It is *not* recommended to `import` a global variable/function,
+since this will force the compiler to interpret fields and methods as `static`
+even if they were intended to be instance.
```

Comment:
Is this a "Do not ever" or a "Try to avoid" guideline?  Let's be clear about that. 

If it is a "Try to avoid", in what case would someone choose to do this?  What is the result of them being resolved as `static`?